src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2288,6 +2306,75 @@ public static IEnumerable<object[]> SystemTimeZonesTestData()
             {
                 yield return new object[] { tz };
             }
+
+            // Include fixed offset IANA zones in the test data when they are available.
+            if (!PlatformDetection.IsWindows)
```

Comment:
>IsWindows [](start = 35, length = 9)

we have static variable s_isWindows used in many places. it is up to you to use it.

Reply:
Is one better than the other performance-wise?

Reply:
not really. it is just the style we used in this file. doing either would be fine. I was just mentioning it to be aware of.

Reply:
Thanks